core/src/main/java/nl/nn/adapterframework/webcontrol/api/ShowInlineMessageStoreOverview.java

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2022 WeAreFrank!
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package nl.nn.adapterframework.webcontrol.api;
+
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import nl.nn.adapterframework.core.Adapter;
+import nl.nn.adapterframework.core.IListener;
+import nl.nn.adapterframework.core.IMessageBrowser;
+import nl.nn.adapterframework.core.IProvidesMessageBrowsers;
+import nl.nn.adapterframework.core.ListenerException;
+import nl.nn.adapterframework.core.ProcessState;
+import nl.nn.adapterframework.receivers.Receiver;
+
+@Path("/")
+public final class ShowInlineMessageStoreOverview extends Base {
+
+	@GET
+	@RolesAllowed({"IbisObserver", "IbisDataAdmin", "IbisAdmin", "IbisTester"})
+	@Path("inlinestores/overview")
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response execute() throws ApiException {
+
+		Map<String, InlineStoreStateItem> storeItemsGroupedByProcessState = new LinkedHashMap<>();
```

Comment:
You can make this a map `Map<ProcessState, InlineStoreStateItem>`

Reply:
Here I just need the names; Done, InProcess... If I pass processState itself it will give me all in upper cased(DONE) then I need to handle in frontend